certbot-dns-route53/certbot_dns_route53/_internal/dns_route53.py

```diff
@@ -32,21 +36,21 @@ class Authenticator(dns_common.DNSAuthenticator):
                    "DNS).")
     ttl = 10
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
         self.r53 = boto3.client("route53")
         self._resource_records: DefaultDict[str, List[Dict[str, str]]] = collections.defaultdict(list)
 
-    def more_info(self):  # pylint: disable=missing-function-docstring
+    def more_info(self) -> str:
         return "Solve a DNS01 challenge using AWS Route53"
 
-    def _setup_credentials(self):
+    def _setup_credentials(self) -> None:
         pass
 
-    def _perform(self, domain, validation_name, validation):
+    def _perform(self, domain: str, validation_name: str, validation: str) -> None:
         pass
 
-    def perform(self, achalls):
+    def perform(self, achalls: Iterable[AnnotatedChallenge]) -> List[ChallengeResponse]:
```

Comment:
Similar thing I guess. Do we want overridden functions to match the annotations of the parent?

```suggestion
    def perform(self, achalls: List[AnnotatedChallenge]) -> List[ChallengeResponse]:
```

Reply:
As above, indeed I prefer to keep the original types.